apiPivotalTest/src/test/resources/cucumber/features/project/updateProject.feature

```diff
@@ -0,0 +1,22 @@
+Feature:Update project
+
```

Comment:
This feature file and deleteProject.feature have the same precondition (Background)
Please join them.